index.html

```diff
@@ -6,6 +6,42 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta name="description" content="PyCon India, the premier conference in India on using and developing the Python programming language is conducted annually by the Python developer community and it attracts the best Python programmers from across the country and abroad.">
   <meta name="author" content="">
+  <!-- Structured Data markup starts -->
+  <script type="application/ld+json">{"@context":"http://schema.org","@type":"Organization","name":"PyCon India","url":"https://in.pycon.org/2018/","logo":"https://in.pycon.org/2018/img/banner-logo.png","sameAs":["https://twitter.com/pyconindia","https://www.facebook.com/PyConIndia/","https://www.linkedin.com/company/pycon-india/","https://www.linkedin.com/groups/3203762"],"areaServed":{"@type":"Country","name":"India"},"contactPoint":[{"@type":"ContactPoint","email":"contact@in.pycon.org","contactType":"Customer Support","url":"https://in.pycon.org/2018/"}]}</script>
```

Comment:
@inovizz Can we keep the json content in separate lines?

Reply:
No, it has to be like this for search engines to crawl the source code, If we move the JSON in some other file then the code would be populated at runtime which search bots don't see/or crawl.

We need to have this available in the source code.

Reply:
Will fix it.

Reply:
Fixed.

Reply:
What I meant by separate lines was not in a separate file. But break the json in separate lines to make it more readable. Can that be done ?

Reply:
It can be done but not needed, as minified JSON or Javascript is always preferable and recommended. This allows search bots to parse content quickly.